services/sn_analysis_service/app.py

```diff
@@ -85,13 +106,18 @@ async def run_sn_model(dd):
         redshift = Table.read(data_dict["inputs"]["redshift"], format='ascii.csv')
         z = redshift['redshift'][0]
     except Exception as e:
-        results.update(
+        rez.update(
             {
                 "status": "failure",
                 "message": f"input data is not in the expected format {e}",
             }
         )
-        upload_analysis_results(results, data_dict)
+        upload_analysis_results(rez, data_dict)
```

Comment:
Doesn't this need to be outside the exception call?

Reply:
Need to see when this is None and return error.

Reply:
refactored so that we use the tornado callback infrastructure to send our results on success or exception/Error